.github/workflows/consistency-checks.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       run: |
         sudo apt update -qq && sudo apt install llvm-dev remake
         python -m pip install --upgrade pip
+        # First, install our Timed-Threads
```

Comment:
@mmatera Now that we have a package that can be installed from PyPI should it be a required package (except when installing from pyodide and in that case dropped), or an optional one where we also install it here?